mender/templates/api-gateway-deploy.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.api_gateway.enabled }}
+{{- $merged := merge (deepCopy .Values.api_gateway) (deepCopy (default (dict) .Values.global)) -}}
```

Comment:
Should we instead create a new `default` section? The `global` values are shared across sub-charts and already contain a lot of attributes that are not used at the service level.

Reply:
I would prefer the first option to avoid potential name collisions.

Reply:
Got it. Let me fix

Reply:
Done, you can review it again